Dockerfile

```diff
@@ -0,0 +1,52 @@
+# Stage 1: Build the application
+FROM maven:3.9.6-amazoncorretto-21 as builder
+
+# Set the working directory in the Maven image
+WORKDIR /app
+
+# Copy the java source files and the pom.xml file into the image
+COPY src ./src
+COPY pom.xml .
+
+# Build the application
+RUN mvn clean package
+
+FROM python:3.11
```

Comment:
Also just to point out (not wrong) I would just think its easier to start with a java 21 image and install python... Instead of the other way around.

Cbioportal DockerFile
https://github.com/cBioPortal/cbioportal/blob/14e00a6c5580700b96152c45f34baf150e652c89/docker/web-and-data/Dockerfile#L29

Reply:
Another reason I think it would be easier to use base image eclipse-temurin:21 is you do not have to worry about having a exact URL

Reply:
yes, it makes sense. I made it this way as I thought we plan to get rid of java loader at some point. But from having couple of calls, I've gotten impression that's the java loader is here to stay for a while.